### PR TITLE
clarify args in custom loss tutorial

### PR DESCRIPTION
**Pull request by martinjankowiak**

### CODE REVIEW

**Comment by martinjankowiak** [comment]
@jpchen while you're at it maybe it's time to push the gplvm tutorial to the website?

File: tutorial/source/custom_objectives.ipynb [hunk 1]
```diff
@@ -214,32 +214,25 @@
...
+    "version": 2
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.6.2"
+   "pygments_lexer": "ipython2",
+   "version": "2.7.13"
```
Comment (jpchen) [inline tutorial/source/custom_objectives.ipynb]: can you revert this by running `make scrub` or changing your ipython kernel to 3.6?

**Comment by jpchen** [comment]
> maybe it's time to push the gplvm tutorial to the website?

if you add it to `index.rst` it will be there when i build it.

File: tutorial/source/custom_objectives.ipynb (hunk 1 above)
Comment (martinjankowiak) [reply tutorial/source/custom_objectives.ipynb]: i scrubbed should be ok to merge

**Pull request merged**